File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    odd_number = 0
    even_number = 0
    for index, num in enumerate(num_list):
        if index % 2 == 1:
            odd_number += num
        else:
            even_number += num
    return max(odd_number, even_number)

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int odd_num =0;
        int even_num = 0;
        int i = 0;
        while(i<num_list.length){
            if(i%2==0){
                even_num += num_list[i];
            }else{
                odd_num +=num_list[i];
            }
            i++;
        }
        if(even_num>odd_num){
            answer=even_num;
        }else{
            answer=odd_num;
        }
        return answer;
    }
}